CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+# TiDB Operator v1.0.6 Release Notes
+
+## v1.0.6 What's New
+
+Action required: Users should migrate the configs in `values.yaml` of previous chart releases to the new `values.yaml` of the new chart. Otherwise, the monitor pods will be failed when upgrading monitor with the new chart.
```

Comment:
```suggestion
Action required: Users should migrate the configs in `values.yaml` of previous chart releases to the new `values.yaml` of the new chart. Otherwise, the monitor pods might fail when you upgrade the monitor with the new chart.
```